pihole-google.txt

```diff
@@ -7399,4 +7404,321 @@ www.tf-enterprise-devel.gcp.fitbit.com
 www.vps.store-au.fitbit.com
 www.wiki.fitbit.com
 www.windchill.fitbit.com
-wwwinternal.fitbit.com
\ No newline at end of file
+wwwinternal.fitbit.com
+android-safebrowsing.googleapis.com
+developers.google.com
+malware-error.mozilla.con
+phish-error.mozilla.con
+safebrowsing.google.com
+safebrowsing.googleapis.com
+sb-ssl.google.com
+sb.l.google.com
+acceleratedmobilepageurl.googleapis.com
+android.bugly.qq.com
+app-measurement.com
+#Duckduckgo
+improving.duckduckgo.com
```

Comment:
Duckduckgo.com domains are not linked to Google. These should be removed

Reply:
All changes made